src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC } from 'react';
+import { TodoFilter } from '../TodoFilter';
+import { Todo } from '../../types/Todo';
+import { FilterTodosBy } from '../../types/FilterTodosBy';
+
+type Props = {
+  todos: Todo[],
+  filterBy: string,
```

Comment:
The `filterBy` prop has a string type, but it is more appropriate to use the `FilterTodosBy` type for consistency and better type checking. 📝